### ISSUE TITLE
Row doesn't position the elements declared by Repeater.

### ISSUE THREAD

**Issue by paiext** [pureqml/qmlcore#179]
```
Rectangle {
  id: root;
  width: 300;
  height: 300;   
  property array colors: ['red','green','blue'];    
  
  function getColor(index) {
    return root.colors[index];
  }

  Row {
    id: row;
    spacing: 5;    
    Repeater {
      model: 3;      
      delegate: Rectangle {
        width: 100;
        height: 100;
        color: root.getColor(model.index);
      }
    }
  }
}
```

![image](https://user-images.githubusercontent.com/5268554/73828220-35741f00-4844-11ea-9d68-e0389324e51e.png)

**Comment by whoozle** [comment]
This is known problem hence Repeater is not layout per-se and it has it's own element inside. I'll try to fix it, but consider that Row + Repeater + Model is effectively ListView + Model :)

**Comment by whoozle** [comment]
Fixed in fc4995c98523a6c1a3a7e623aaefaa4dfe0ca401
In case of any problem feel free to reopen or create new issue :)

**Comment by paiext** [comment]
@whoozle alright. thank you 👌
